.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   e2e:
     docker:
-    - image: cimg/go:1.19 # If you update this, update it in the Makefile too
+    - image: cimg/go:1.20 # If you update this, update it in the Makefile too
```

Comment:
It's available now

https://hub.docker.com/layers/cimg/go/1.20.0/images/sha256-2a6480070739d5c0739b7b7784e90a2250d5f654a2ee6ec5911860fd24bb8f91?context=explore

Reply:
💯